synapse/storage/schema/delta/56/signing_keys.sql

```diff
@@ -13,6 +13,47 @@
  * limitations under the License.
  */
 
+-- cross-signing keys
```

Comment:
this needs to go in a separate file to the `devices` update, so that they can be merged to `develop` independently.